cns/saas/admin_guide/start/enforcer/enforcer.adoc

```diff
@@ -1,26 +1,297 @@
-//weight: 40
-//menu:
-//  saas:
-//    parent: "start"
-//    identifier: "deploy-enforcer"
-//canonical: https://docs.aporeto.com/saas/start/enforcer/
-//aliases: [
-//  "/saas/reference/components/enforcer/"
-//]
-//
-//'''
-
-. Ensure that your target host or cluster meets the xref:reqs.adoc[system requirements].
-. If there is a non-transparent proxy in the data path between your Enforcers and the Prisma Cloud SaaS console, you must enable support for a xref:./transparent-proxy.adoc[non-transparent proxy].
-. Refer to the appropriate installation procedure.
-+
-* xref:k8s.adoc[Kubernetes/OpenShift]
-* xref:linux.adoc[Linux hosts]
-* xref:windows.adoc[Windows hosts]
+== Install the Enforcer
+
+You can install Enforcers on bare metal servers or virtual machines (VMs) running on private clouds or on-prem. 
+
+The following procedures cover the Enforcer installation in these environments. 
+
+=== Prerequisites
+
+Review the xref:../reqs.adoc[system requirements].
+Deploying Enforcers requires that namespaces be present in your environment.
+If you don't have any suitable namespaces in your environment, xref:../create-ns.adoc[create the namespaces] before deploying Enforcers.
+
+[.task]
+=== Linux
+
+[.procedure]
+
+. Navigate to the *Namespace* of the group where you want to deploy the Enforcer. From there, go to the *Agent > Deploy* page.
+
+. Select *Single Enforcer*.
+
+. Set the *Platform Type* to *Linux*.
+
+. Set the *Registration Type* to *Standard*.
++
+image::install-enforcer-1.png[width=300,align="center"]
+
+. If you want to add custom tags to your Enforcer use them in a `RuleSet`, add them under *Tags*.
++
+image::install-enforcer-2.png[width=300,align="center"]
+
+. Copy the resulting installation script and run it on your VMs to install the Enforcer.
++
+image::install-enforcer-3.png[width=300,align="center"]
+
+[.task]
+=== Windows Servers and Clients
+
+[.procedure]
+
+. Navigate to the *Namespace* of the group where you want to deploy the Enforcer. From there, go to the *Agent > Deploy* page.
+
+. Select *Single Enforcer*.
+
+. Set the *Platform Type* to *Windows*.
+
+. Set the *Registration Type* to *Standard*.
++
+image::install-enforcer-4.png[width=300,align="center"]
+
+. If you want to add custom tags to your Enforcer use them in a `RuleSet`, add them under *Tags*.
++
+image::install-enforcer-2.png[width=300,align="center"]
+
+. Copy the resulting installation script and run it on your servers and clients to install the Enforcer.
++
+image::install-enforcer-5.png[width=300,align="center"]
+
+=== Install Enforcers on Public Cloud Instances with Auto-Registration
+
+The difference between an Enforcer deployed on-prem and an Enforcer deployed on a public cloud is the registration method.
+
+On-prem Enforcers use a short-lived microsegmentation token.
+Cloud resources using Cloud Auto-Registration use a token signed by the cloud provider, which allows the Enforcer to retrieve cloud provider metadata.
+
```

Comment:
When installing the Enforcer on Cloud Providers, we recommend you to use the Cloud AutoRegistration method.
This will make sure the Enforcer uses the short-lived tokens and that allows the Enforcer to retrieve the cloud provider metadata. Any new VM that has the cloud IAM role attached to it, and has the enforcer binary installed, will be automatically registered to the namespace.

Reply:
@rjain15 Thank you for providing the reasons to use auto-registration. I've added the information. PTAL.